evm/rpc/modules/eth.py

```diff
@@ -20,51 +45,52 @@ class Eth(RPCModule):
     def accounts(self):
         raise NotImplementedError()
 
+    @apply_to_return_value(hex)
```

Comment:
Given the simplicity of this function, can we just change the return statement to be `hex(num)` and avoid the albeit minor overhead of the additional function call.